### ISSUE TITLE
Support for analytics in configuration

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-gui#199]
It should be possible in the configuration to add a custom block of HTML near the foot of every page. This would give us an easy way to use analytics tools that don't disappear when you reinstall or upgrade Zonemaster GUI. That way we could add something like [Goat Counter](https://www.goatcounter.com/#features) (a more privacy minded alternative to Google Analytics) to zonemaster.net so we can get a better picture of how it's being used.

**Comment by pamasse** [comment]
Since the index.html page is the main page (Single Page Application), you only need to put the code once.

**Comment by mattias-p** [comment]
Hello @pamasse! Nice to see you around :)

But index.html is overwritten when you reinstall Zonemaster GUI or upgrade it to a newer version, and then you have to update it again, right?

**Comment by pamasse** [comment]
Yes, a technical solution will be to supercharge the starting script with a "user configuration loading script".
This script will inject the javascript and HTML code associated with the analytical solution. 

Another solution could be to implement a generic solution such as https://github.com/angulartics/angulartics2, and pre-configure the library with specific metrics such as, how many people launch a test, how many read the FAQ pages, how many visitors change the language, etc. It more time expensive.
